clients/lcdproc/disk.c

```diff
@@ -124,32 +137,71 @@ disk_screen(int rep, int display, int *flags_ptr)
 	sock_printf(sock, "widget_set D f 1 2 %i %i %i %i v 12\n", lcd_wid, lcd_hgt, lcd_wid, count);
 	for (i = 0; i < count; i++) {
 		char tmp[lcd_wid + 1];	/* should be large enough */
+		char cap[8] = {'\0'};
```

Comment:
I'm honestly not good enough with C to know, if this is equivalent to the memset() code we had before?

Reply:
Yes, brace initializers will zero-fill unspecified elements, e.g. from CPPReference on C array initialization: 

`int z[3] = {0}; // z has type int[3] and holds all zeroes`

Reply:
Just for the benefit of my education: Then `char cap[8] = {};` would work as well?

Reply:
In C++, yes - but I do not think an empty initializer is allowed in to-spec C (though some compilers might support it anyway)